### PR TITLE
SW-1350 Add collectors to accession payloads

### PR DESCRIPTION
**Pull request by sgrimm**

### CODE REVIEW

**Comment by sgrimm** [comment]
Current dependencies on/for this PR:
* main
  * **PR #534** <a href="https://app.graphite.dev/github/pr/terraware/terraware-server/534" target="_blank">Graphite</a> 
    * **PR #535** <a href="https://app.graphite.dev/github/pr/terraware/terraware-server/535" target="_blank">Graphite</a> 
      * **PR #536** <a href="https://app.graphite.dev/github/pr/terraware/terraware-server/536" target="_blank">Graphite</a> 
        * **PR #537** <a href="https://app.graphite.dev/github/pr/terraware/terraware-server/537" target="_blank">Graphite</a>  👈
          * **PR #538** <a href="https://app.graphite.dev/github/pr/terraware/terraware-server/538" target="_blank">Graphite</a> 
            * **PR #539** <a href="https://app.graphite.dev/github/pr/terraware/terraware-server/539" target="_blank">Graphite</a> 
              * **PR #540** <a href="https://app.graphite.dev/github/pr/terraware/terraware-server/540" target="_blank">Graphite</a> 

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/terraware/terraware-server/537?utm_source=stack-comment).

File: src/main/kotlin/com/terraformation/backend/seedbank/api/AccessionsController.kt [hunk 1]
```diff
@@ -130,11 +130,30 @@ class AccessionsController(
   }
 }
 
+/**
+ * Returns the list of collectors from an incoming request payload that might be coming from a
+ * client that doesn't know about the new unified `collectors` field. The client will remove the old
+ * fields when it starts using the new one, so if the old fields are present, use them and ignore
+ * the new one.
+ */
+private fun backwardCompatibleCollectors(
```
Comment (karthikbtf) [inline src/main/kotlin/com/terraformation/backend/seedbank/api/AccessionsController.kt:139]: thanks

**Review by karthikbtf — APPROVED** [review]
Thanks for the changes per our discussion.

**Comment by sgrimm** [comment]
**Graphite Merge Job**

Current status: ✅ Merged

This pull request was successfully merged as part of a stack.

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/terraware/terraware-server/537?utm_source=stack-comment).


Job Reference: XRvz5790CdoSGIEGLvxb

**Pull request merged**